source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -2257,12 +2327,13 @@ These errors indicate one of these:
 * A read was attempted on an unreadable server
   (arbiter, ghost, or recovering)
   or a read was attempted on a read-only server without the slaveOk bit set.
+* An operation was attempted on a server that is now shutting down.
 
 In any case the error is a symptom that
 a ServerDescription's type no longer reflects reality.
 
-A primary closes its connections when it steps down,
-so in many cases the next operation causes a network error
+On MongoDB 4.0 and earlier, a primary closes its connections when it steps
+down, so in many cases the next operation causes a network error
```

Comment:
This is some misc. outdated prose I discovered while reading the spec but it's otherwise unrelated to this PR.